packages/react/src/VideoPlayer/VideoPlayer.tsx

```diff
@@ -70,7 +70,7 @@ const Root = ({
   return (
     <div className={styles.VideoPlayer__container} ref={fullscreenRef}>
       <div className={styles.VideoPlayer__overlayContainer}>
-        <video ref={ref} title={title} controls={false} className={clsx(styles.VideoPlayer, className)} {...rest}>
+        <video title={title} controls={false} className={clsx(styles.VideoPlayer, className)} {...rest} ref={ref}>
```

Comment:
@joshfarrant can you add a test for this to ensure the ref doesn't get overridden?

Reply:
I've just tested in react-live's demo and the `ref` prop isn't being set to `null` there, so maybe there's a bug in our implementation. I'll put this PR on hold for a while and look into it a bit further